examples/energy_mng/src/main.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+#![cfg_attr(feature = "mcu-board-support", no_std)]
+#![cfg_attr(all(feature = "mcu-board-support", not(feature = "simulator")), no_main)]
+
+#[cfg(feature = "mcu-board-support")]
+extern crate alloc;
+
+#[cfg(target_arch = "wasm32")]
+use wasm_bindgen::prelude::*;
+
+#[allow(clippy::all)]
```

Comment:
Are you testing with clippy?

It would be better to actually fix clippy warnings in the generated code.
Maybe by adding that `allow` thing in the generated code itself

Reply:
I use it so in other projects. And other people will look the example codes and copy paste stuff and maybe check their code with clippy. And therefore I think it is good to do so until the clippy warnings in the generated code are fixed. I also like that you so reference the ui stuff with `ui` e.g. `ui::MainWindow`.

Reply:
IMHO this shouldn't be in our example.
The example should be as clean as possible from hacks.
We should just have generated code that doesn't generate warnings.